scripts/globals/mobskills/carousel.lua

```diff
@@ -9,7 +9,8 @@ require("scripts/globals/mobskills")
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
-    if mob:isMobType(xi.mobskills.mobType.NOTORIOUS) then
+    if mob:isMobType(xi.mobskills.mobType.NOTORIOUS) or
+    mob:isMobType(xi.mobskills.mobType.BATTLEFIELD) then
```

Comment:
LSB just implemented a new check that if statements need multiple lines that they should be formatted like:

```
    if 
        mob:isMobType(xi.mobskills.mobType.NOTORIOUS) or
        mob:isMobType(xi.mobskills.mobType.BATTLEFIELD)
    then
```

Reply:
updated.